integration/create_rules_docker_buildfile.rb

```diff
@@ -0,0 +1,6 @@
+require 'erb'
```

Comment:
Thanks for adding an example!

This file right here has the unfortunate side effect of adding a BIG dependency on a CI server at shops that don't have ruby.

What would you think about replacing this with a bash script that uses a [heredoc](https://linuxize.com/post/bash-heredoc/)

```
base_path="$(git rev-parse --show-toplevel)/integration"
packages="$1"
cat <<EOF > "${base_path}/BUILD.bazel"
load("@io_bazel_rules_docker//container:container.bzl", "container_image")
load("@io_bazel_rules_docker//docker/util:run.bzl", "container_run_and_commit_layer")
load("@io_bazel_rules_docker//docker/package_managers:download_pkgs.bzl", "download_pkgs")
load("@io_bazel_rules_docker//docker/package_managers:install_pkgs.bzl", "install_pkgs")

# Base docker image for our Scala services
download_pkgs(
    name = "base_packages",
    image_tar = "@openjdk_11_slim//image",
    packages = [
        "${packages}",
    ],
)
... then the rest
EOF

Reply:
@achew22 I will work in this PR on removing all ruby, thanks for the heredoc